README.md

```diff
@@ -1,2 +1,14 @@
 # hw01
```

Comment:
Reviewed by Sarah J

Reply:
Sarah- You used many of the elements suggested in the rubric including headers, lists, links, and pictures and did successfully upload your markdown file (I read your comments). In addition, your document is clean and easy to read. I noticed one small typo when you introduce your picture (perhaps missing "of"); however, this does not interfere with any of the coding. I might suggest finding a way to scale your picture so it's not so big. This is something I encountered when uploading my own photo, and unfortunately, I have not yet found a solution. Additionally, I like how you included in-text links. I didn't know you could attach links this way. Your approach is creative and makes your document easy to navigate. Overall, nice job!